src/Psalm/Internal/Codebase/AssertionsFromInheritanceResolver.php

```diff
@@ -61,80 +57,9 @@ public function resolve(
              * Since the inheritance does not provide its own assertions, we have to detect those
```

Comment:
Seems like this comment should change too?

Reply:
well Actually we still have to detect those, but in this case we just merge. 
any suggestions? I have no clue how to modify this for being more accurate 🫤

Reply:
nope! I was just making sure there was not an outdated comment left behind :)